test/serverless.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// mock of the serverless instance
+class Serverless {
```

Comment:
This copies the serverless stub from the [Google plugin](https://github.com/serverless/serverless-google-cloudfunctions/blob/master/test/serverless.js)